packages/slice-machine/src/features/onboarding/OnboardingGuide.tsx

```diff
@@ -0,0 +1,29 @@
+import { Card, CardContent, Text } from "@prismicio/editor-ui";
+
+import { useOnboardingProgress } from "@/features/onboarding/helpers";
+import { OnboardingProgressStepper } from "@/features/onboarding/OnboardingProgressStepper";
+
+export const OnboardingGuide = () => {
+  const { steps, completedStepCount, isComplete } = useOnboardingProgress();
+
+  if (isComplete) return null;
+
+  return (
+    <Card color="grey2" variant="outlined" paddingBlock={16}>
+      <CardContent>
+        <div>
```

Comment:
I could do that, but since `Text` renders a span, it will appear inline by default. I could create a class to apply `display: block` to the parent Text. Not sure I prefer having that, but I also don't like unnecessary nesting just for styling, so what do you prefer?

Reply:
not important, you can leave it like this